### PR TITLE
Feature/acceptance mock refactor

### PR DESCRIPTION
**Pull request by jcalderin**
Implement --> https://github.com/telefonicaid/fiware-orion/issues/763

"NOT MERGE YET, REFACTORING IN PROCESS"

### CODE REVIEW

**Comment by jcalderin** [comment]
https://github.com/telefonicaid/fiware-orion/commit/add064f1ec2f9a0c4253e0c91b5f431991fa3941#commitcomment-9735142 fixed in https://github.com/telefonicaid/fiware-orion/commit/94913b27cb44c38833253a57830213f956fa7c13

**Comment by fgalan** [comment]
LGTM (alghouth I think that the LGTM which is important in this PR is the one from @iariasleon )

**Comment by iariasleon** [comment]
LGTM

**Comment by jcalderin** [comment]
Looking for improve the mock, a mock from "lettuce_tool", a TID framwork (https://github.com/telefonicaid/lettuce-tools), is available, then i will do a refactor to use it, and not re-invent the wheel... :)

File: test/acceptance/integration/tools/mock.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+"""
+
+__author__ = 'Jon Calderin Goñi (jon.caldering@gmail.com)'
+
+# from flask import Flask, request
```
Comment (iariasleon) [inline test/acceptance/integration/tools/mock.py]: delete these lines of the old mock

File: test/acceptance/properties.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
     "port": "1026"
   },
   "mock": {
-    "port": "5566"
+    "port": "5566",
```
Comment (iariasleon) [inline test/acceptance/properties.json]: Change this value to 8000 (Port where to start the mock (8000 by default)

**Pull request closed without merging**